src/Init/Data/List/Basic.lean

```diff
@@ -164,13 +170,19 @@ theorem isEqv_cons₂ : isEqv (a::as) (b::bs) eqv = (eqv a b && isEqv as bs eqv)
 
 /-! ## Lexicographic ordering -/
 
-/-- Lexicographic ordering for lists. -/
+/--
+Lexicographic ordering for lists with respect to an ordering of elements.
+-/
 inductive Lex (r : α → α → Prop) : List α → List α → Prop
   /-- `[]` is the smallest element in the order. -/
   | nil {a l} : Lex r [] (a :: l)
-  /-- If `a` is indistinguishable from `b` and `as < bs`, then `a::as < b::bs`. -/
+  /--
+  Two lists with the same head, but ordered tails, are ordered.
+  -/
   | cons {a l₁ l₂} (h : Lex r l₁ l₂) : Lex r (a :: l₁) (a :: l₂)
-  /-- If `a < b` then `a::as < b::bs`. -/
+  /--
+  If the head of the first list is smaller than the head of the second, then they are ordered.
```

Comment:
Similarly. How about "then the first list is lexicographically smaller than the second".